login_V1

```diff
@@ -1,37 +1,32 @@
+
 <?php
-	ob_start();
-	session_start();
-		$error = NULL;
-		if($_SERVER["REQUEST_METHOD"] == "POST") {
-			// connect.php (tells where to connect servername, dbaseName, username, password)
-			require "Database_mysqli.php";
-			// username and passwrod sent from form 
-			$myusername = mysqli_real_escape_string($conn, $_POST['username']);
-			$mypassword = mysqli_real_escape_string($conn, $_POST['password']);
-			
-			$query = "SELECT Username FROM Users WHERE Username = '$myusername' and Password = '$mypassword'";
-			
-			$result = mysqli_query($conn, $query);
-			$row = mysqli_fetch_array($result, MYSQLI_ASSOC);
+	$error = NULL;
+	if($_SERVER["REQUEST_METHOD"] == "POST") {
 			
-			$count = mysqli_num_rows($result);
-			
-			// if result matched $my_username and $mypassword, table row must be 1 row
-			if($count == 1) {
-				$_SESSION['login_user'] = $myusername;
-				header("location: index.html");
-			} else {
-				$error = "Invalid Credentials. Please Try Again";
-				}
+		require "Database_mysqli.php";
+		
+		$myusername = $_POST['username'];
+
+		$insertquery = "DELETE FROM Users WHERE Username = '$myusername'";
+
+		if (mysqli_query($conn, $insertquery))
+			{
+			$error = "User Successfully Thrown Into The Void";
+			$color = '#3d9c46';
 		}
-	ob_end_flush();
+		else 
+		{
+			$error = "Error: Username Does Not Exist";
+			$color = '#d44e6b';
+		}
+	}
```

Comment:
Here is some additional error catching in case the username that the user entered does not exist.